### PR TITLE
Hw 1

### PR DESCRIPTION
**Pull request by MikhaylovVladimir**
Первое обязательно домашнее задание по Deep learning

### CODE REVIEW

**Review by pacifikus — CHANGES_REQUESTED** [review]
Добрый день!

Спасибо за работу! В целом все хорошо! 

Оставлю несколько комментариев о том, что можно улучшить:
- Для предотвращения лика пайплайн применения `StandardScaler` и некоторых других трансформаций выглядит как "разбиение на train/test -> fit_transform на train -> transform на test"
- Пока что в работе нет выводов относительно выбранной архитектуры/гиперпараметров/оптимизатора/преобразований и т.п. - это был один из пунктов задания
- Также отсутствуют эксперименты с расположением слоев `BatchNorm1d` и `Dropout` в архитектуре и выводы об их влиянии на сходимость
- Эксперименты или обоснование значения параметра `p` для `Dropout` также отсутствуют
